From d01d54d76fe2f3cd2c3c598aeaaf4788be01f30c Mon Sep 17 00:00:00 2001
From: Fang Wang <fang.c.wang@intel.com>
Date: Thu, 27 Nov 2014 20:42:18 +0800
Subject: [PATCH 8/8] atomisp2: Changing sensor setting

Removed one sensor setting
to resolve the bug about video and preview has FOV issue.

Change-Id: I674cee325968431a32eba571f27677b5b99de462
Tracked-On: https://jira01.devtools.intel.com/browse/IMINAN-6224
Signed-off-by: Fang Wang <fang.c.wang@intel.com>
Reviewed-on: https://icggerrit.ir.intel.com/30471
Approver: Wei Tao <wei.tao@intel.com>
Build: Wei Tao <wei.tao@intel.com>
Reviewed-by: Wei Tao <wei.tao@intel.com>
Maintainer: Wei Tao <wei.tao@intel.com>
---
 .../camera/drivers/media/i2c/ov2722.h              |   16 ----------------
 1 file changed, 16 deletions(-)

diff --git a/drivers/external_drivers/camera/drivers/media/i2c/ov2722.h b/drivers/external_drivers/camera/drivers/media/i2c/ov2722.h
index 3b6d57f..0ea0033 100644
--- a/drivers/external_drivers/camera/drivers/media/i2c/ov2722.h
+++ b/drivers/external_drivers/camera/drivers/media/i2c/ov2722.h
@@ -1271,22 +1271,6 @@ struct ov2722_resolution ov2722_res_video[] = {
 		.mipi_freq = 364800,
 	},
 	{
-		.desc = "ov2722_1M3_30fps",
-		.width = 1416,
-		.height = 1034,
-		.fps = 30,
-		.pix_clk_freq = 73,
-		.used = 0,
-		.pixels_per_line = 2048,
-		.lines_per_frame = 1184,
-		.bin_factor_x = 1,
-		.bin_factor_y = 1,
-		.bin_mode = 0,
-		.skip_frames = 3,
-		.regs = ov2722_1M3_30fps,
-		.mipi_freq = 364800,
-	},
-	{
 		.desc = "ov2722_1080P_30fps",
 		.width = 1932,
 		.height = 1092,
-- 
1.7.9.5

